homeassistant/components/wyoming/config_flow.py

```diff
@@ -112,3 +99,52 @@ async def async_step_hassio_confirm(
             description_placeholders={"addon": self._hassio_discovery.name},
             errors=errors,
         )
+
+    async def async_step_zeroconf(
+        self, discovery_info: zeroconf.ZeroconfServiceInfo
+    ) -> FlowResult:
+        """Handle zeroconf discovery."""
+        _LOGGER.debug("Discovery info: %s", discovery_info)
+        if discovery_info.port is None:
+            return self.async_abort(reason="no_port")
+
+        service = await WyomingService.create(discovery_info.host, discovery_info.port)
+        if (service is None) or (not (name := service.get_name())):
+            return self.async_abort(reason="no_services")
+
+        self.context[CONF_NAME] = name
+        self.context["title_placeholders"] = {"name": name}
+
+        uuid = f"wyoming_{service.host}_{service.port}"
```

Comment:
This is not a valid uuid. If it's just used to find if host/port has been configured, manually iterate the entries.